### ISSUE TITLE
nothing was moved, many errors about file size and also "TypeError: cannot use a bytes pattern on a string-like object" at end

### ISSUE THREAD

**Issue by owldown** [gtronset/beets-filetote#111]
I just did a huge import and tried the filetote plug-in with this in my config.yaml:

```
filetote:
  extensions: .*
```

I thought that would move all non-music files.

Instead, nothing moved, and throughout the import (in which I was using -q and -I, auto tagging and moving to a different folder), I got this error on what appears to be the first track of each album (here is one example):
```
could not get filesize: [Errno 2] No such file or directory: b'/Users/knewman/beetsmusicsurely/music2023/Music/Yo La Tengo/Painful/01 Big Day Coming.mp3'
```

Then, at the end of the import,

```
Traceback (most recent call last):
  File "/Users/knewman/beetsmusicsurely/venv/bin/beet", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beets/ui/__init__.py", line 1285, in main
    _raw_main(args)
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beets/ui/__init__.py", line 1274, in _raw_main
    plugins.send('cli_exit', lib=lib)
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beets/plugins.py", line 488, in send
    result = handler(**arguments)
             ^^^^^^^^^^^^^^^^^^^^
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beets/plugins.py", line 145, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beetsplug/filetote.py", line 420, in process_events
    self.process_artifacts(
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beetsplug/filetote.py", line 576, in process_artifacts
    artifact_dest = util.unique_path(artifact_dest)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/knewman/beetsmusicsurely/venv/lib/python3.11/site-packages/beets/util/__init__.py", line 602, in unique_path
    match = re.search(br'\.(\d)+$', base)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/lib/python3.11/re/__init__.py", line 176, in search
    return _compile(pattern, flags).search(string)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a bytes pattern on a string-like object
```

**Comment by gtronset** [comment]
@owldown The first issue mentioned (`could not get filesize`) should be resolved with `v0.4.3`. I'll start taking a look at the secondary issue here soon. Can you please confirm the `beet` command that was run (exactly, if possible)?

**Comment by gtronset** [comment]
@owldown The issue `TypeError: cannot use a bytes pattern on a string-like object` should be resolved with `v0.4.4`. Can you please retest and confirm?
